packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -3,7 +3,9 @@ import { IconButton } from '../../Button';
 import { Label } from '../../Label';
 import { Icon } from '../../Icon';
 import {
-  BaseComponent,
+  initializeComponentRef,
```

Comment:
In regards to `initializeFocusRects` that was mentioned by @KevinTCoughlin in your #9826 PR, I think this component could be used as an individual component in an app where no other Fabric components are used so you might be missing the focus rectangles unless because `SpinButton` is using `IconButton`, `Label` and `Icon` fabric controls that might be initializing it instead. Like Kevin mentioned this should be resolved systematically by providing guidance in our documentation to have that `initializeFocusRects` call once in the beginning of their app like we do with icons, but for now to be on the safe side I would include that call.